azure/v14.0.0/release.yaml

```diff
@@ -0,0 +1,63 @@
+# Generated with:
+# devctl release create --name 14.0.0 --base v13.1.1 --releases . --provider azure --overwrite true --component kubernetes@1.19.4 --component app-operator@3.0.0 --component cluster-operator@0.23.20 --component azure-operator@5.2.1-dev --component containerlinux@2605.10.0 --component etcd@3.4.14 --app cluster-autoscaler@1.19.1  --app azure-scheduled-events@0.1.0
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v14.0.0
+spec:
+  apps:
+    - name: cert-exporter
+      version: 1.5.0
+    - name: chart-operator
+      version: 2.6.0
+    - componentVersion: 1.6.5
+      name: coredns
+      version: 1.2.0
+    - componentVersion: 0.7.4
+      name: external-dns
+      version: 1.5.0
+    - componentVersion: 1.9.7
+      name: kube-state-metrics
+      version: 1.3.0
+    - name: metrics-server
+      version: 1.2.1
+    - name: net-exporter
+      version: 1.9.2
+    - componentVersion: 1.0.1
+      name: node-exporter
+      version: 1.7.1
+    - name: cluster-autoscaler
+      version: 1.19.1
+    - name: azure-scheduled-events
+      version: 0.1.0
+  components:
+    - name: app-operator
+      releaseOperatorDeploy: true
+      version: 3.0.0
+    - name: azure-operator
+      catalog: control-plane-test-catalog
+      releaseOperatorDeploy: true
+      version: 5.2.2-dev
+      reference: 5.2.0-9b3aea291daacdc6f61ad67c9a91c1163be347d1
+    - name: cert-operator
+      reference: 0.1.0-2
+      releaseOperatorDeploy: true
+      version: 0.1.0
+    - name: cluster-operator
+      releaseOperatorDeploy: true
+      version: 0.23.20
+    - name: kubernetes
+      version: 1.19.4
```

Comment:
Is there a specific reason to go with `1.19.4` when latest one is `1.19.7` ?

Reply:
IIRC when we started only 1.19.4 was out - 1.19.7 should be fine now

Reply:
Yeah, there were bunch of bugfixes for Azure disk related stuff (again) and few fixes for OOM / CPU burn situations.

Reply:
wlll bump it

Reply:
done